src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -86,6 +86,13 @@
                              :KhtmlUserSelect "none" :MsUserSelect "none"}} props)
    children])
 
+(defn create-link2 [{:keys [href style onClick text]}]
+  [:a (merge
+        {:href (or href "javascript:;")
+         :style (merge {:textDecoration "none" :color (:button-blue colors)} style)}
+        (when onClick {:onClick onClick}))
+   text])
+
 (defn create-link [onClick & children]
```

Comment:
I'm actually just going to replace all usages of the old one with the new one (and then rename).  But that would touch almost every UI page file.